src/stories/UI/MenuTrigger.examples.tsx

```diff
@@ -132,3 +144,42 @@ export const MultipleControlled = () => {
     </>
   );
 };
+
+export const CustomInlineMenu = () => {
+  return (
+    <MenuTrigger
+      portalSelector="#portal"
+      onOpenChange={(isOpen) => console.log("isOpen:", isOpen)}
+      // isOpen
+      hideArrow
+      placement="right"
+      offset={20}
+      popupClassName={triggerExample.inlineMenuPopup}
+    >
+      <Button
+        tone={2}
+        variant="fab"
+        vol={2}
+        icon={<AddIcon alt="Add item" />}
+      />
+      <Menu
+        selectionMode="none"
+        onAction={(value) => console.log(value)}
+        className={triggerExample.inlineMenu}
+      >
+        <Item key="addImage">
+          <AddImageIcon alt="Add image" />
+        </Item>
+        <Item key="addVideo">
+          <AddVideoIcon alt="Add video" />
+        </Item>
+        <Item key="addPDF">
+          <AddPDFIcon alt="Add PDF" />
```

Comment:
This could be AddDocumentIcon (probably using a different icon) to allow for different types of documents not only PDF.